src/applications/vaos/services/appointment/transformers.js

```diff
@@ -183,8 +183,8 @@ export function transformVAOSAppointment(appt) {
       vistaId: appt.locationId?.substr(0, 3) || null,
       clinicId: appt.clinic,
       stationId: appt.locationId,
-      clinicName: appt.serviceName || null,
-      clinicPhysicalLocation: appt.physicalLocation || null,
+      clinicName: appt.serviceName,
+      clinicPhysicalLocation: appt.physicalLocation,
```

Comment:
No need for Object.hasOwnProperty call:

```
// Include clinicName only if property exists
...(Object.prototype.hasOwnProperty.call(appt, 'serviceName') && {
  clinicName: appt.serviceName || null,
}),
```
The desired results are achieved with an just an assignment. The 'clinicName' and 'clinicPhysicalLocation' attributes are set as follows:

When appt.serviceName does not exist: clinicName = undefined
When appt.serviceName exist but value is empty string or null: clinicName = null
When appt.serviceName exist and not null: clinicName = value